### ISSUE TITLE
[Suggestion/Request] Buttons to Set Preferred Subtitle to be at the top when Joining 2 subtitles

### ISSUE THREAD

**Issue by MbuguaDavid** [SubtitleEdit/subtitleedit#5392]
Is it possible to have buttons to set the preferred subtitle when joining 2 or more subtitles?

Normally, when you select 2 subtitles from a folder and add them when joining subtitles, the first subtitle in the list will be the second subtitle in the preview window. 

To avoid this, one needs to add one subtitle at a time knowing the first subtitle will be at the bottom and the second subtitle will be at the top. 

Adding buttons to determine which subtitle will be on top would be very helpful and save time. 
![Buttons when Joining Subtitles](https://user-images.githubusercontent.com/92556454/137416279-b024eedd-57c5-430c-9de1-eb49ed5622f3.png)

Thanks!

**Comment by cyzs233** [comment]
This is the default behavior of how subtitle lines with same timecode stacks.
![stack subtitles](https://user-images.githubusercontent.com/25927091/153726070-d9201da1-2a73-4140-993e-82f125c652c5.jpg)

if you want to display in opposite way, you can do an extra stack using "tool/merge line with same time code"
![extra stack subtitles](https://user-images.githubusercontent.com/25927091/153726174-be47ef36-b4d6-4bdd-a3b4-32a0f2de3759.jpg)

Anyway, Button to adjust input order is a pretty convenient option.

**Comment by cyzs233** [comment]
By the way, I encountered an related problem and had a look at this issue one more time. 
From what I have gathered (having fun guessing the internal logic), subtitleedit will fetch the **Start time** of the first subtitle line from each file and happily reorder your files for you in this **Join subtitles** Window.
![Untitled](https://user-images.githubusercontent.com/25927091/178437706-8798b430-d394-43cb-8384-04e88bfb8557.png)

Since this "Forcing file order base on Start time" is an odd thing to do in the first place, plus, niksedk still hasn't removed this not-so-wanted reordering feature yet, I assume there will be severe changes involved. 
So I don't think your request is happening any time soon.
